hack/run-e2e-conformance-virtual-cluster.sh

```diff
@@ -361,19 +361,35 @@ do
 done
 
 
+source hack/env.sh
+
 export ADMISSION_CONTROLLERS_ENABLED=true
 export ADMISSION_CONTROLLERS_CERTIFICATES_CERT_MANAGER_ENABLED=true
 export SKIP_VAR_SET=""
 export NAMESPACE="sriov-network-operator"
 export OPERATOR_NAMESPACE="sriov-network-operator"
 export CNI_BIN_PATH=/opt/cni/bin
 export OPERATOR_EXEC=kubectl
-export CLUSTER_TYPE=kubernetes
-export DEV_MODE=TRUE
 export CLUSTER_HAS_EMULATED_PF=TRUE
 
-echo "## deploy namespace"
-envsubst< $root/deploy/namespace.yaml | ${OPERATOR_EXEC} apply -f -
+
+HELM_VALUES_OPTS="\
+  --set images.operator=${SRIOV_NETWORK_OPERATOR_IMAGE} \
+  --set images.sriovConfigDaemon=${SRIOV_NETWORK_CONFIG_DAEMON_IMAGE} \
+  --set images.sriovCni=${SRIOV_CNI_IMAGE} \
+  --set images.sriovDevicePlugin=${SRIOV_DEVICE_PLUGIN_IMAGE} \
+  --set images.resourcesInjector=${NETWORK_RESOURCES_INJECTOR_IMAGE} \
+  --set images.webhook=${SRIOV_NETWORK_WEBHOOK_IMAGE} \
+  --set operator.admissionControllers.enabled=${ADMISSION_CONTROLLERS_ENABLED} \
+  --set operator.admissionControllers.certificates.certManager.enabled=${ADMISSION_CONTROLLERS_CERTIFICATES_CERT_MANAGER_ENABLED} \
+  --set sriovOperatorConfig.deploy=true"
+
+PATH=$PATH:${root}/bin
+make helm
+helm  install -n ${NAMESPACE} --create-namespace \
+  $HELM_VALUES_OPTS \
+  --wait sriov-network-operator ./deployment/sriov-network-operator
```

Comment:
FYI #706 will rename the chart to sriov-network-operator-chart.

Reply:
Ack!